pkg/kubelet/kubelet.go

```diff
@@ -2846,13 +2853,15 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod) *v1.Pod {
 		// Update pod resource allocation checkpoint
 		if err := kl.statusManager.SetPodAllocation(updatedPod); err != nil {
 			//TODO(vinaykul,InPlacePodVerticalScaling): Can we recover from this in some way? Investigate
+			kl.recorder.Eventf(updatedPod, v1.EventTypeWarning, events.ResizeFailed, "Resize pod %s failed: %v", klog.KObj(updatedPod), err)
 			klog.ErrorS(err, "SetPodAllocation failed", "pod", klog.KObj(updatedPod))
 			return pod
 		}
 	}
```

Comment:
Understand, because the Resize process is relatively long, maybe we should send events at key points. Events should not be like logs, but a key feedback to trigger "state" changes.